arch/inst/Smrnmi/mnret.yaml

```diff
@@ -3,9 +3,9 @@
 $schema: inst_schema.json#
 kind: instruction
 name: mnret
-long_name: No synopsis available.
+long_name: Machine NMI Return
```

Comment:
Where did this text come from?

What I find is that the "mnret" instruction is defined by "Smrnmi" extension for resumable non-maskable interrupts. And these textual references to this particular instruction:
> MNRET, to resume from the RNMI handler.

> MNRET is an M-mode-only instruction [...] to return to the program counter, privilege mode, and virtualization mode of the interrupted context.

... no great references.

M = Machine. Fine.

Given the first reference, and specifically the use of "RNMI", perhaps the "long_name" should be something like "Machine Resume from RNMI handler".